src/.env

```diff
@@ -0,0 +1,3 @@
+USER_INVALID_PASSWORD='inval';
+USER_VALID_PASSWORD='validPassword';
```

Comment:
Why are you setting testing values as environment variables? Those should be just constants in the test file or a fixture.